spec/models/zip_endpoint_spec.rb

```diff
@@ -132,7 +134,7 @@
     end
 
     describe '.which_need_archive_copy' do
-      let(:names) { [ep.endpoint_name, zip_endpoint.endpoint_name] }
+      let(:names) { [ep.endpoint_name, ep2.endpoint_name, zip_endpoint.endpoint_name] }
```

Comment:
In that case, I'm seeing:
```ruby
  1) ZipEndpoint ZippedMoabVersion presence on ZipEndpoint .which_need_archive_copy returns the zip endpoints which should have a complete moab for the druid/version, but which don't yet
     Failure/Error: expect(ZipEndpoint.which_need_archive_copy(druid, version).pluck(:endpoint_name).sort).to eq %w[mock_archive1 zip-endpoint]
     
       expected: ["mock_archive1", "zip-endpoint"]
            got: ["ibm_us_south", "zip-endpoint"]
     
       (compared using ==)
     # ./spec/models/zip_endpoint_spec.rb:148:in `block (4 levels) in <top (required)>'
     # ./spec/support/live_s3.rb:8:in `block (2 levels) in <top (required)>'
```

Reply:
I feel wary about this case, since there's not just a different order of elements in what we got, but different elements than expected when making the change.

Reply:
paired w/ @tallenaz on this just now, turns out the `#order` in the query matters because we create ZMVs on `other_ep1` etc, and if `other_ep1` isn't deterministic, the expected endpoints that still need a copy in subsequent expectations will be more of a pain to figure out.  so leaving this as-is, since the `#sort` on the list of names doesn't seem particularly consequential or worth an extra commit.